--- origsrc/Client.build	2007-01-11 16:00:24.000000000 -0600
+++ src/Client.build	2007-06-25 22:23:34.906250000 -0500
@@ -34,8 +34,7 @@
 	</if>
 
 	<!-- builds the client on all supported frameworks -->
-	<target name="allframeworks" depends="net-1.1">
-		<call target="Docs"/>
+	<target name="allframeworks" depends="mono-1.0">
 	</target>
 
 	<!-- core managed driver target -->	
--- origsrc/Samples/TableEditor/CS/TableEditor.build	2007-01-09 10:28:40.000000000 -0600
+++ src/Samples/TableEditor/CS/TableEditor.build	2007-06-25 22:47:44.718750000 -0500
@@ -12,9 +12,7 @@
 	</if>
 
 	<target name="default">
-		<property name="framework" value="net-1.0"/>
-		<call target="base"/>
-		<property name="framework" value="net-1.1"/>
+		<property name="framework" value="mono-1.0"/>
 		<call target="base"/>
 	</target>	
 
--- origsrc/mysqlclient/AssemblyInfo.cs	2007-02-01 15:14:58.000000000 -0600
+++ src/mysqlclient/AssemblyInfo.cs	2007-06-25 22:53:58.000000000 -0500
@@ -78,7 +78,5 @@
 //       documentation for more information on this.
 //
 [assembly: AssemblyDelaySign(false)]
-#if FINAL
-[assembly: AssemblyKeyFile("")]
-[assembly: AssemblyKeyName("ConnectorNet")]
-#endif
+[assembly: AssemblyKeyFile("../../../../mysql.snk")]
+[assembly: AssemblyKeyName("")]
